arm-machinelearning/2016-05-01-preview/swagger/commitmentPlans.json

```diff
@@ -0,0 +1,720 @@
+        },
+        {
+          "name": "resourceGroupName",
+          "description": "The resource group name.",
+          "in": "path",
+          "required": true,
+          "type": "string"
+        },
+        {
+          "name": "planName",
+          "description": "The commitment plan name.",
+          "in": "path",
+          "required": true,
+          "type": "string"
+        },
+        {
+          "$ref": "#/parameters/APIVersionParameter"
+        }
+      ]
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.MachineLearning/commitmentPlans": {
+      "get": {
+        "tags": [
+          "CommitmentPlans"
+        ],
+        "operationId": "CommitmentPlans_List",
+        "description": "Retrieve all Azure ML commitment plans in a subscription.",
+        "parameters": [
+          {
+           "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "$skipToken",
+            "description": "Continuation token for pagination.",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/PaginatedResult[CommitmentPlan]"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearning/commitmentPlans": {
+      "get": {
+        "tags": [
+          "CommitmentPlans"
+        ],
+        "operationId": "CommitmentPlans_ListInResourceGroup",
+        "description": "Retrieve all Azure ML commitment plans in a resource group.",
+        "parameters": [
+          {
+           "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "description": "The resource name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "$skipToken",
+            "description": "Continuation token for pagination.",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/PaginatedResult[CommitmentPlan]"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearning/commitmentPlans/{planName}/usageHistory": {
+      "get": {
+        "tags": [
+          "CommitmentPlans"
+        ],
+        "operationId": "CommitmentPlans_GetUsageHistory",
```

Comment:
This is more of a list operation then a get. It is pageable as well. How about naming this as `Usages_List`? Is there any other component than the CommitmentPlan in this swagger spec that the user can get usage of?